--- ./usr/lib/plexmediaserver/lib/plexmediaserver.service.old	2021-02-09 11:37:39.645726259 +0100
+++ ./usr/lib/plexmediaserver/lib/plexmediaserver.service	2021-02-09 11:38:33.776846948 +0100
@@ -27,7 +27,8 @@
 export PLEX_MEDIA_SERVER_INFO_VENDOR="$(grep ^NAME= /etc/os-release | awk -F= "{print \\$2}" | tr -d \\" )"; \
 export PLEX_MEDIA_SERVER_INFO_DEVICE="PC"; \
 export PLEX_MEDIA_SERVER_INFO_MODEL="$(uname -m)"; \
-export PLEX_MEDIA_SERVER_INFO_PLATFORM_VERSION="$(grep ^VERSION= /etc/os-release | awk -F= "{print \\$2}" | tr -d \\" )"; \
+export GENTOO_PROFILE="`readlink /etc/portage/make.profile`"; \
+export PLEX_MEDIA_SERVER_INFO_PLATFORM_VERSION="$${GENTOO_PROFILE#.*/profiles/}"; \
 export LD_LIBRARY_PATH=/usr/lib/plexmediaserver/lib; \
 exec "/usr/lib/plexmediaserver/Plex Media Server"'
 Type=simple
